mscore/exportmidi.cpp

```diff
@@ -79,6 +79,33 @@ void ExportMidi::writeHeader()
             }
 #endif
 
+      //--------------------------------------------
+      //    write track and instrument names
```

Comment:
This should be just “write track names” for now, right?

Reply:
Yes, and maybe better to change the comment.

Reply:
Which I just did. Thanks!